src/TypeScript.UnitTests/Analyzer/JavaScriptAnalyzerTests.cs

```diff
@@ -73,6 +74,72 @@ public void IsAnalysisSupported_HasJavascript_True()
             result.Should().BeTrue();
         }
 
+        [TestMethod]
+        public async Task ExecuteAnalysis_AlwaysStartsEslintBridgeServer()
+        {
+            var serverProcess = SetupServerProcess(123);
+            var client = SetupEslintBridgeClient(null);
+            var clientFactory = SetupEslintBridgeClientFactory(123, client.Object);
+            
+            var testSubject = CreateTestSubject(clientFactory, serverProcess.Object);
+
+            await testSubject.ExecuteAnalysis("some path", Mock.Of<IIssueConsumer>(), CancellationToken.None);
+
+            serverProcess.Verify(x => x.Start(), Times.Once);
+            serverProcess.VerifyNoOtherCalls();
+        }
+
+        [TestMethod]
+        public async Task ExecuteAnalysis_FailsToStartsEslintBridgeServer_ExceptionCaughtAndLogged()
```

Comment:
This test might be redundant, we already have `ExecuteAnalysis_NonCriticalException_ExceptionCaughtAndLogged` and `ExecuteAnalysis_CriticalException_ExceptionThrown`, so I'm not sure it adds much value.

Reply:
It's not adding a lot from a technical point of view. It is testing a specific end-user scenario i.e. a known type of failure, but it's not adding to the overall test coverage.
I probably wouldn't keep it, but I don't have a strong feeling about it.